coalib/misc/Constants.py

```diff
@@ -128,39 +128,3 @@
     r'(?::\d+)?'  # optional port number
     r'(?:/?|[/?]\S+)$',  # path
     re.IGNORECASE)
-
-
-def configure_logging():
-    """
-    Configures the logging with hard coded dictionary.
-    """
-    import sys
-    import logging.config
-    logging.config.dictConfig({
-        'version': 1,
-        'handlers': {
-            'colored': {
-                'class': 'logging.StreamHandler',
-                'formatter': 'colored',
-                'stream': sys.stderr
-            }
-        },
-        'root': {
-            'level': 'DEBUG',
-            'handlers': ['colored']
-        },
-        'formatters': {
-            'colored': {
-                '()': 'colorlog.ColoredFormatter',
-                'format': '%(log_color)s[%(levelname)s]%(reset)s[%(asctime)s]'
-                          ' %(message)s',
-                'datefmt': '%X',
-                'log_colors': {
-                    'ERROR': 'red',
-                    'WARNING': 'yellow',
-                    'INFO': 'blue',
-                    'DEBUG': 'green'
-                }
-            }
-        }
-    })
```

Comment:
sorry, most people don't think of reset and add -p I think it's not in any of our tutorials even so I figured it might save you some time to mention it ;)